core/coreapi/name.go

```diff
@@ -75,7 +81,7 @@ func (api *NameAPI) Publish(ctx context.Context, p path.Path, opts ...caopts.Nam
 	}
 
 	if options.TTL != nil {
-		ctx = context.WithValue(ctx, "ipns-publish-ttl", *options.TTL)
```

Comment:
Breaking with:

https://github.com/ipfs/go-namesys/blob/ba24a61c15763fbe30f939c18bcbf3823d43ae1b/publisher.go#L314

Tip to find thoses:
```
go-ipfs$ go mod vendor
go-ipfs$ grep -r "ipns-publish-ttl"
```